src/lib/cfa/ntp_conf.rb

```diff
@@ -99,15 +99,27 @@ def fix_keys(tree)
 
     # class to manage ntp entries as a collection.
     #
-    # Each element of the collection is a subclass of Record.
-    # A Record object is a wrapper of an AugeasElement.
+    # The collection preserves the order of the
+    # ntp entries in the file and only contains the
+    # entries of interest (see RECORD_ENTRIES).
+    #
+    # For example:
+    #
+    # server 1.pool.ntp.org iburst
+    # server 2.pool.ntp.org
+    # peer 128.100.0.45
+    #
+    # The collection contents a derived Record class
+    # object for each line. In this case, two
+    # ServerRecord and a PeerRecord.
     class RecordCollection
       include Enumerable
 
       def initialize(augeas_tree)
         @augeas_tree = augeas_tree
       end
 
+      # Iterates the elements of the collection.
```

Comment:
and add what parameter yield have http://www.rubydoc.info/gems/yard/file/docs/Tags.md#yield